scripts/logextract.py

```diff
@@ -489,9 +487,9 @@ def parse_line(self, line_num, line):
     def get_lines(self):
         # Ignore old stats
         all_ts = []
-        for mcu_name, mcu in self.mcus.items():
-            all_ts.extend(mcu.sent_seq_to_time.values())
-            all_ts.extend(mcu.receive_seq_to_time.values())
+        for mcu_name, mcu in list(self.mcus.items()):
+            all_ts.extend(list(mcu.sent_seq_to_time.values()))
```

Comment:
`list.extend()` accepts an iterable and does the right thing. Also no need to convert to list.